.github/workflows/lint-test.yml

```diff
@@ -0,0 +1,73 @@
+name: Lint and Test
+
+# Run this when someone pushes a branch
```

Comment:
> Run this when someone pushes a branch

Does this mean it'll run only once when we first push the branch? or should this be saying "commit" instead?

Reply:
It will only run when you `push` to a branch on the remote.  So you can commit all you want locally, but when you `push` to origin is when this will run.

Reply:
ah gotcha.. okay.. so pushing to a branch. Makes sense 👍🏼